src/views/Entities/Dataplanes.vue

```diff
@@ -625,9 +642,22 @@ export default {
                 return data
               }
 
+              const getDpStatus = async () => {
+                try {
+                  const res = await this.$api.getDataplaneOverviewFromMesh(entityMesh, entity.name)
```

Comment:
Correct me if I'm wrong, but I see that we get a Dataplane in 596, then for this dataplane we've get overview at 606 and then overview again here. 3 requests. Can we make just one request for overview in 596?

Reply:
Yeah, I know, the initial idea was not to touch many lines with this PR, but it appears not so much of a work, so I fixed that. Please take look